### DISCUSSION TITLE
封号原因小调查

### DISCUSSION THREAD

**Discussion by lss233** [lss233/chatgpt-mirai-qq-bot#362] [General]
最近封号的情况越来越多了，想知道大家的账号是怎么搞的导致的封号？    
被封的和没被封的都可以进来聊聊。  
```  
使用的模式：（Plus？Web、API、还是都有用？）
聊天话题：（专业问题咨询、日常聊天、涩涩？）
使用的 endpoint:
聊天频率：
版本：
账号来源：（买的号？自己注册的？）  
使用的IP地区：
注册IP地区：
手机号地区：
```

**Comment by xiaohai1222** [comment]
用web模式   放到群里边 我朋友聊天   聊天评论很快 一条没结束下一条就来了  openai经常提示一小时后在访问

自己注册的账号

**Reply by lss233** [reply]
这和封号没什么关系啊

**Reply by xiaohai1222** [reply]
openai的号被封了啊    endpoint = https://bypass.duti.tech/  我朋友和我用的同样的配置 号也没了  他和我一样也是那机器人放我们群里  然后没了

**Reply by lss233** [reply]
提示一小时后访问 和封号无关。

**Reply by xiaohai1222** [reply]
前两天一直会触发提示一小时后访问  今天在登录openai发现号就已经被封了

**Comment by Qianmo2** [comment]
plus，存活。
使用的模式：Web
聊天话题：专业问题咨询+聊天
使用的 endpoint:群主自己的
聊天频率：100条/天
版本：2.2.3.1
账号来源：群主代购plus
使用的IP地区：美国，纽约
注册IP地区：
手机号地区：

**Comment by tinygeeker** [comment]
使用的模式：Web
聊天话题：专业问题咨询
使用的 endpoint: https://bypass.duti.tech/
聊天频率：超低，估计2条/小时
版本：v2.2.3.1
账号来源：自己注册的 
使用的IP地区：美国
注册IP地区：
手机号地区：

**Comment by jijunhao** [comment]
使用的模式：都用无plus，今天挂了两个号 1.web+api 2.web
聊天话题：专业问题咨询+日常聊天
使用的 endpoint: 群主的
聊天频率：50-100/每天
版本：2.2.3.1
账号来源：自己注册的  qq邮箱 
使用的IP地区：日本
注册IP地区：日本
手机号地区：印度sms虚拟手机号
寄！！！！！！哭

**Comment by zhangshujie02** [comment]
使用的模式：使用邮箱登录web
聊天话题：一些机械的专业问题和日常聊天
使用的 endpoint:3月14日使用的是配置文件默认endpoint，今天3月15日早上换了lss233的endpoint
聊天频率：基本白天早上和下午 60%
版本：2.2.3.1
账号来源：自己用谷歌邮箱账号注册的
使用的IP地区：美国
注册IP地区：美国
手机号地区：美国

**Reply by lss233** [reply]
请不要公开我的 endpoint

**Reply by zhangshujie02** [reply]
> 请不要公开我的 endpoint

ok

**Comment by hus-king** [comment]
使用的模式：都有用
聊天话题：专业问题咨询
使用的 endpoint:  来自群主
聊天频率：一天100条左右
版本：最新
账号来源：自己注册的
使用的IP地区：美国硅谷
注册IP地区：美国硅谷
手机号地区：巴西

**Comment by Kabisang** [comment]
使用的模式：web 
聊天话题：大部分专业问题
使用的 endpoint:一开始是默认的 后面换成另一个了
聊天频率：十几条/一小时
版本：2.2.3
账号来源：自己注册的号和买的号同时使用
使用的IP地区：HK
注册IP地区：
手机号地区：

**Comment by superjond** [comment]
使用的模式：Web browserless
聊天话题：专业问题咨询
使用的 endpoint:bypass.duti.tech/
聊天频率：十几条/一小时
版本：2.23
账号来源：淘宝  
使用的IP地区：美国
注册IP地区：
手机号地区：

**Comment by evilne** [comment]
两个    号1      号2 
使用的模式：Web browserless
聊天话题：日常聊天
使用的 endpoint: 默认https://bypass.duti.tech/  
聊天频率：低
版本：2.2.3
账号来源：1.买的outlook   2.qq邮箱手动  
使用的IP地区：代理在 日本、美国切换
注册IP地区：
手机号地区：2.印度

1. web 的api从未使用 ,昨晚挂机器人的 今天没的
2 .  使用号1之前一直web browserless 挂机器人，前天本地使用ChatPaper，挂了代理用API ,5分钟内收到邮件被封

**Comment by LayerLwb** [comment]
使用的模式：Web
聊天话题：专业问题咨询、日常聊天
使用的 endpoint:无
聊天频率：1天3条
版本：最新版
账号来源：自己注册
使用的IP地区：美国
注册IP地区：美国
手机号地区：阿根廷

**Comment by LayerLwb** [comment]
使用的模式：Web
聊天话题：专业问题咨询、日常聊天
使用的 endpoint:https://bypass.duti.tech/api/
聊天频率：1天10条以内
版本：最新版
账号来源：自己注册
使用的IP地区：美国
注册IP地区：美国
手机号地区：印度尼西亚

**Reply by vinson-lee01** [reply]
Expect support

**Comment by saily21** [comment]
使用的模式：web
聊天话题：专业问题+正常闲聊
使用的 endpoint: bypass.duti.tech/
聊天频率：100/天
版本：2.2.3
账号来源：自己注册
使用的IP地区：日本、美国
注册IP地区：日本
手机号地区：sms巴西
两个普通一个plus全寄了（裂开）

**Comment by lorewalkerpan** [comment]
使用的模式：都有用,非plus，api和web共用同一个账号
聊天话题：专业问题咨询
使用的 endpoint:群主搭建
聊天频率：20左右/天
版本：2.2.3
账号来源：自己注册的gmail邮箱
使用的IP地区：日本，美国
注册IP地区：美国
手机号地区：美国
早上9点半左右进了下官网，发现号被封，调用api接口发现能用，再刷新重新调用，返回就是号被封了，再去看了下web，web也封了

**Comment by FullscreenError** [comment]
使用的模式：web、key、bing
聊天话题：日常以及专业问题
使用的 endpoint:默认
聊天频率：不高
版本：2.2.3.1
账号来源：web(自己号)、key(买的号)、bing(自己号)
使用的IP地区：硅谷
注册IP地区：美国
手机号地区：web号(巴西)、key号(美国)
注：key是在3.13封的，web是在3.15封的，bing目前正常，配置基本默认，服务器IP是硅谷的，无代理，系统Ubuntu18
